### ISSUE TITLE
Compatibility with Jquery 3.0

### ISSUE THREAD

**Issue by mapolonio** [rochal/jQuery-slimScroll#257]
Hi, I would like to note that as Jquery 3.0 was released I've found that some migration issues arise.

Using the Jquery migration plugin I get this warning:
`jQuery.fn.bind() is deprecated`

I hope we can see an update for this in the future.

**Comment by baktun14** [comment]
I just used '.on' instead of '.bind' and it works the same

**Comment by mapolonio** [comment]
Yes, of course. But it's not recommended to modify external libraries because in the future someone may just update from the repository and will generate conflicts. That's why it's expected that the author aknowledge the changes needed to be made.
